src/styles/blocks/form.scss

```diff
@@ -0,0 +1,38 @@
+.form {
+  &__field {
+    width: 100%;
+    height: 48px;
+    display: block;
+    font-family: inherit;
+    font-size: 14px;
+    padding: 14px 16px;
+    line-height: 143%;
+    margin-bottom: 16px;
+    color: #292929;
+    background-color: #f8f8fa;
+
+    border: 1px solid transparent;
+    border-radius: 40px;
+
+    &::placeholder {
+      color: grey;
+    }
+
+    &:hover {
+      border-color: #d5d7de;
+    }
+
+    &:focus {
+      border-color: #292929;
```

Comment:
These too

Reply:
I see only 1 usecase of these colors, why we need to make them variables?